src/gl/tessellation_helpers.cpp

```diff
@@ -37,10 +36,8 @@ mgl::Primitive mgl::tessellate_renderable_into_rectangle(
     mgl::Primitive rectangle;
     rectangle.type = GL_TRIANGLE_STRIP;
 
-    GLfloat tex_right = static_cast<GLfloat>(rect.size.width.as_int()) /
-                        buf_size.width.as_int();
-    GLfloat tex_bottom = static_cast<GLfloat>(rect.size.height.as_int()) /
-                         buf_size.height.as_int();
```

Comment:
The old logic attempts to keep buffer size the same regardless of the size of the rectangle the buffer is being rendered to. The new logic stretches the buffer to the full are of the rectangle. Scaling is achieved by having a rectangle a different size than the buffer.